src/main/scala/com/codacy/client/bitbucket/v2/service/UserServices.scala

```diff
@@ -12,21 +12,22 @@ class UserServices(client: BitbucketClient) {
    * Gets the basic information associated with the token owner account.
    */
   def getUser: RequestResponse[User] = {
-    client.execute(Request("https://api.bitbucket.org/2.0/user", classOf[User]))
+    client.execute(Request(client.userBaseUrl, classOf[User]))
   }
 
   /*
-   * Gets the basic information associated with an account.
+   * Gets all the emails of an account
    */
-  def getUser(userId: String): RequestResponse[User] = {
-    client.execute(Request(s"https://api.bitbucket.org/2.0/users/$userId", classOf[User]))
+  def getEmails: RequestResponse[Seq[Email]] = {
+    client.executePaginated(Request(s"${client.userBaseUrl}/emails", classOf[Seq[Email]]))
   }
 
   /*
-   * Gets all the emails of an account
+   * Gets the basic information associated with an account.
    */
-  def getEmails: RequestResponse[Seq[Email]] = {
-    client.executePaginated(Request(s"https://bitbucket.org/api/2.0/user/emails", classOf[Seq[Email]]))
+  def getUser(userId: String): RequestResponse[User] = {
+    val encodedUserId = URLEncoder.encode(userId, "UTF-8")
```

Comment:
Codacy found an issue: [The string literal 'UTF-8' appears 2 times in the file.](https://app.codacy.com/gh/codacy/bitbucket-scala-client/pullRequest?prid=5907044)